src/core/src/type/float8_e4m3.cpp

```diff
@@ -9,6 +9,7 @@
 #include <limits>
 
 #include "openvino/core/type/float_util.hpp"
+#include "openvino/reference/fake_convert.hpp"
```

Comment:
Why this include is required ?
Or just float17 should be included here?

Reply:
Applied. Thanks Pawel!